src/components/gds-button/_index.scss

```diff
@@ -0,0 +1,54 @@
+@mixin base {
```

Comment:
Why did you use _index.scss instead of let's say mixins.scss?

Reply:
So you can do `@import '~genero-design-system/src/components/gds-button` rather than `@import '~genero-design-system/src/components/gds-button/mixins`.

Reply:
I get your point, but personally I like that it says what it is. If this is a file for just mixins, then it's clearer that it's mixins.js.

But that's just me, and I'm not writing that code, so do what you think is best. :)